examples/platform/silabs/BaseApplication.cpp

```diff
@@ -74,6 +74,13 @@
 #include <performance_test_commands.h>
 #endif // PERFORMANCE_TEST_ENABLED
 
+// SL-Only
+#include "sl_component_catalog.h"
```

Comment:
Just wanted to double check that you are intentionally including "sl_component_catalog.h" for all apps here?

Reply:
Yes, I want to use `SL_CATALOG_ZIGBEE_STACK_COMMON_PRESENT` and the include only brings existing definitions to the BaseApplication.cpp. No consequence for other apps